src/createBaseRouter.js

```diff
@@ -56,6 +56,19 @@ export default function createBaseRouter({ render }) {
       if (!this.props.initialRenderArgs) {
         this.resolveMatch();
       }
+
+      if (__DEV__ && typeof window !== 'undefined') {
```

Comment:
just curious, why this instead of a simpler `&& window`?

Reply:
javascripts

```
> window
ReferenceError: window is not defined
> typeof window !== 'undefined'
false
```